ci/matrix.yaml

```diff
@@ -84,10 +87,49 @@ pull_request:
     - {cuda: *cuda_curr,     os: 'windows2022', cpu: 'amd64', compiler: *msvc2019, std: [14, 17],         jobs: ['build']}
     - {cuda: *cuda_curr,     os: 'windows2022', cpu: 'amd64', compiler: *msvc2022, std: [14, 17, 20],     jobs: ['build']}
     - {cuda: *cuda_curr,     os: 'ubuntu22.04', cpu: 'amd64', compiler: *oneapi,   std: [11, 14, 17],     jobs: ['build']}
+  nvcc-cuda-next:
```

Comment:
We can prune this down if we decide to tighten up the minimum host compiler versions, etc.